examples/chapter1/annexb/src/annexb5.cpp

```diff
@@ -44,6 +44,8 @@ int main(int argc, char *argv[])
                                                                              poolSize,
                                                                              initializerFunction,
                                                                              eventClonerFunction);
+    auto eventLoopPublisher = (kpsr::high_performance::EventLoopPublisher<TestEvent, 4> *)
```

Comment:
prefer using c++ style casts --> `auto eventLoopPublisher = std::static_cast<kpsr::high_performance::EventLoopPublisher<TestEvent, bufferSize> >(kpsrPublisher)`. (Note that in the code your bufferSize for provider was 16 and here you have used 4.